paasta_tools/oom_logger.py

```diff
@@ -136,11 +151,15 @@ def capture_oom_events_from_stdin():
                 break
 
 
-def get_container_env_as_dict(docker_inspect):
+def get_container_env_as_dict(is_cri_containerd: bool, container_inspect):
```

Comment:
nit:
```suggestion
def get_container_env_as_dict(is_cri_containerd: bool, container_inspect: dict) -> Dict[str, str]:
```

comment: typing `container_inspect` is probably a PITA if the containerd/docker libraries don't have a good TypedDict/dataclass/etc that we can use - maybe once we're only using containerd we can have a non-total TypedDict with the bits that we do use tho :p